nautilus-extension/src/nautilus_module.rs

```diff
@@ -139,7 +139,7 @@ impl NautilusModule {
         };
 
         unsafe {
-            let module_type = g_type_module_register_type(self.module, G_TYPE_OBJECT, name.as_ptr(), &info, GTypeFlags::empty());
+            let module_type = g_type_module_register_type(self.module, G_TYPE_OBJECT, name.as_ptr(), &info, G_CONNECT_AFTER);
```

Comment:
I don't think `G_CONNECT_AFTER` is a valid flag for `g_type_module_register_type`:

https://developer.gnome.org/gobject/unstable/GTypeModule.html#g-type-module-register-type

https://developer.gnome.org/gobject/unstable/gobject-Type-Information.html#GTypeFlags

`G_TYPE_FLAG_ABSTRACT` and `G_TYPE_FLAG_VALUE_ABSTRACT` are allowed.

Reply:
Yes, this is correct. My bad. I'm not sure which value to use though. I'm guessing that `G_TYPE_FLAG_VALUE_ABSTRACT` would be correct here though.